libs/event-manager/web-client/features/list-events/src/lib/search-filter.pipe.ts

```diff
@@ -0,0 +1,15 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { EventItem } from '@event-manager-web-client/data-access';
+@Pipe({
+  name: 'searchFilter',
+})
+export class SearchFilterPipe implements PipeTransform {
+  transform(value: EventItem[], search: string): EventItem[] | undefined {
+    const regexp = new RegExp(search, 'i');
```

Comment:
Does this regular expression do something besides checking that "search" is included in "value"?

Reply:
no, do you think it's wrong?